packages/npm-packages/ruby-wasm-wasi/test/unit/test_js.rb

```diff
@@ -27,4 +27,11 @@ def test_eval
   def test_try_convert
     assert_nil JS.try_convert(Object.new)
   end
+
+  def test_constasts
+    assert_equal 'null', JS::Null.to_s
+    assert_equal 'undefined', JS::Undefined.to_s
+    assert_equal 'true', JS::True.to_s
+    assert_equal 'false', JS::False.to_s
```

Comment:
If I use `JS::Object` with `assert_equal` as the argument, I get the following error:

> JS::Error: TypeError: Cannot read properties of null (reading 'encoding')

To avoid the error, I compare with a string.